lib/reducers/create-otp-reducer.js

```diff
@@ -162,7 +162,7 @@ export function getInitialState(userDefinedConfig) {
       sort: {
         direction: 'ASC',
         // Only apply custom sort if batch routing is enabled.
-        type: 'BEST'
+        type: config?.itinerary?.defaultSort || 'BEST'
```

Comment:
`config` is defined at line 90 so you don't need to write `config?`.

Reply:
Good catch thanks!